channel-manager-operations/operations.md

```diff
@@ -0,0 +1,23 @@
+# API Operations (CHM)
+
+This section describes all operations supported by the Channel Manager side API, organised here by theme.
+
+## Inventory
+
+| <div style="width:200px">Operation or Endpoint</div> | Description |
```

Comment:
Is fixed width correct ? Doesnt it suport rather percentage ?

Reply:
That's a good point. I don't like putting HTML into the MD, but I made an exception here because it is the only way to make it look decent when you have multiple similar tables. And px or %, I'm not sure as both have pros and cons. I think on balance that px is best but maybe the amount could be better - I will review it.

Reply:
Having reviewed this, I'm inclined not to change it. 200px seems to render well, and still seems to be responsive to smaller browser windows.